whitesands/code/datums/components/crafting/recipes.dm

```diff
@@ -145,3 +145,25 @@
 	time = 20
 	tools = list(TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
 	category = CAT_MISC
+
+/datum/crafting_recipe/ZipPistol
```

Comment:
Change these names to snake_case please

Reply:
I think its resolved now, emphasis on the word think.